### DISCUSSION TITLE
Customizing Area Bump Chart Colors Based on Ending Date in Data Series

### DISCUSSION THREAD

**Discussion by shellyscheng** [plouc/nivo#2556] [Q&A]
Hi, I'm trying to use the area bump chart with custom arrays of colors: ["colorA", "colorB", "colorC", ..., "colorZ"]. How can I assign colors based on the ending date (2006's data in screenshot below) instead of the starting date (2000's data)? 

![Screenshot 2024-04-02 at 3 01 13 PM](https://github.com/plouc/nivo/assets/22440246/748c5b20-3e72-475d-a53e-b29234b708ed)

**Comment by plouc** [comment]
@shellyscheng, there's no straightforward way of doing this, what you could do is to first sort the data by the value for 2006, the colors aren't assigned according to the first values, but according to the order of the input.
